recipe/meta.yaml

```diff
@@ -84,6 +82,9 @@ requirements:
     - hdf5 * {{ mpi_prefix }}_*
     - libblas
     - liblapack
+    # temporary dependency until https://github.com/conda-forge/lapack-feedstock/issues/83
+    # gets resolved
+    - mkl  # [linux and x86_64]
```

Comment:
I'm not happy about having to do this but until things get fixed in `liblapack`, I don't see a way around this.

Reply:
I'm confused. Is the LAPACK dependency causing runtime failures due to how it gets linked in? Can we fix this by changing how MOAB configures/links against the LAPACK library? Strictly speaking, MOAB only requires Eigen3 dependency (but LAPACK is nice and supercedes Eigen3 in places). However, TempestRemap does not have that option and has a mandatory dependency on LAPACK. No way around that.

Reply:
mkl gets linked in at build time in liblapack, which makes it a runtime dependency. U believe that is a mistake but until it is fixed, everything that depends on liblapack at build time needs mkl at runtime.